ts/components/conversation/right-panel/overlay/OverlayRightPanelSettings.tsx

```diff
@@ -251,7 +254,7 @@ const DeleteGroupPanelButton = () => {
     <PanelIconButton
       text={window.i18n(token)}
       dataTestId="leave-group-button"
-      onClick={() => void showLeaveGroupByConvoId(convoId, selectedUsername)}
+      onClick={() => void showDeleteGroupByConvoId(convoId, selectedUsername)}
```

Comment:
Why has this one been changed?

Reply:
Actually I guess because you want to make sure this is a group 100% and also not send a leave message? 

I would have thought we would want to send a leave message?

Reply:
we don't always want to send one. If we want to send one, that component (DeleteGroupPanelButton) won't be rendered but another one will. Does that make sense?

Reply:
Yup it does. Resolving